openpathsampling/pathmover.py

```diff
@@ -877,6 +890,16 @@ def legal_sample_set(globalstate, ensembles=None, replicas='all'):
         self.replicas and the parameter ensembles. If ensembles is None, we
         use self.ensembles. If you want all ensembles allowed, pass
         ensembles='all'.
+
+        Parameters
+        ----------
+        globalstate : `openpathsampling.SampleSet`
```

Comment:
don't worry about changing it before merge, but I think we want to use the syntax ``:class:`.SampleSet` ``. This leads to Sphinx to link correctly. I don't think we need to explicitly label the namespace if it is in the same namespace (makes readability harder, and doesn't add anything for Sphinx).

See: http://www.sphinx-doc.org/en/stable/markup/inline.html#xref-syntax
Example in use: http://openpathsampling.org/latest/api/generated/openpathsampling.pathsimulator.FullBootstrapping.html#openpathsampling.pathsimulator.FullBootstrapping (except the `TISTransition` parameter)